.conan/test_package/CMakeLists.txt

```diff
@@ -4,4 +4,5 @@ project(PackageTest CXX)
 find_package(Catch2 CONFIG REQUIRED)
 
 add_executable(test_package test_package.cpp)
-target_link_libraries(test_package Catch2::Catch2WithMain)
\ No newline at end of file
+target_link_libraries(test_package Catch2::Catch2WithMain)
+target_compile_features(test_package PRIVATE cxx_std_14)
```

Comment:
Conan 1.x does not pass cppstd, neither -std= to the compiler, so we need to define what standard is required to be consume the package. In Conan 2.x there is no such problem because cppstd is always define in the profile.